plugins/phrase-conector/src/phrase.ts

```diff
@@ -34,11 +34,12 @@ export class Phrase {
   }
 
   async request(path: string, config?: RequestInit, search = {}) {
+    let privateKey = await appState.globalState.getPluginPrivateKey(pkg.name);
```

Comment:
This would be expensive, since it would hit firestore everytime instead of once per load. I think that's the reason why it is cached.

Could we look at how many times this URL is called daily to understand impact?